apps/saleor-app-checkout/backend/payments/providers/adyen/createPayment.ts

```diff
@@ -27,48 +27,48 @@ export const createAdyenPayment = async (data: OrderFragment, redirectUrl: strin
 
   const checkout = new CheckoutAPI(client);
 
-  const total = data.total.gross;
+  const total = order.total.gross;
 
   const { url, id } = await checkout.paymentLinks({
```

Comment:
Maybe we could move this boilerplate to a helper function to make this more easily readable?